<ui version="4.0" >
 <class>WindowTriggerWidget</class>
 <widget class="QWidget" name="WindowTriggerWidget" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" >
   <property name="spacing" >
    <number>0</number>
   </property>
   <property name="margin" >
    <number>0</number>
   </property>
   <item>
    <widget class="QGroupBox" name="when_group" >
     <property name="title" >
      <string>Trigger When</string>
     </property>
     <layout class="QGridLayout" name="gridLayout" >
      <property name="margin" >
       <number>4</number>
      </property>
      <item row="0" column="0" >
       <widget class="QRadioButton" name="window_appears" >
        <property name="text" >
         <string>Window Appears</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1" >
       <widget class="QRadioButton" name="window_disappears" >
        <property name="text" >
         <string>Window Disappears</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" >
       <widget class="QRadioButton" name="window_gets_focus" >
        <property name="text" >
         <string>Window Gets Focus</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1" >
       <widget class="QRadioButton" name="window_lost_focus" >
        <property name="text" >
         <string>Window Loses Focus</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="window_group" >
     <property name="title" >
      <string>Window</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
